runtime-library/libdyntype/dynamic-simple/dyn-value/class/date.c

```diff
@@ -6,39 +6,137 @@
 #include "dyn_class.h"
 #include <time.h>
 
+/**
+ * @brief char* to time_t
+ *
+ * @param str year-month-day hour:minute:second, like
+ * "2023-12-23 15:58:12"
+ * @param t time_t addr
+ * @return int 0: success
+ */
+int strtotime(const char *str, time_t *t) {
+  int year, month, day, hour, minute, second;
+  struct tm tm_;
```

Comment:
Had better use clang-format to format the code

Reply:
Done